pkg/compiler/debug.go

```diff
@@ -94,9 +98,10 @@ type DebugRange struct {
 
 // DebugParam represents variables's name and type.
 type DebugParam struct {
-	Name   string                  `json:"name"`
-	Type   string                  `json:"type"`
-	TypeSC smartcontract.ParamType `json:"-"`
+	Name     string                  `json:"name"`
+	Type     string                  `json:"type"`
+	RealType binding.Override        `json:"-"`
+	TypeSC   smartcontract.ParamType `json:"-"`
```

Comment:
Don't we have some duplication in this structure now?

Reply:
No:
- `Type` is a type from `smartcontract/`
- `TypeSC` is a type from `vm/stackitem/`
- `RealType` is a type from the `Go` code.